docs/changelog.rst

```diff
@@ -2,6 +2,7 @@
 Changelog
 =========
 
+* :feature:`-` Add ``/channels/<token_address>`` REST-API endpoint to query all node's channels for a specific token
```

Comment:
please make an issue describing the need and addition for the new rest api endpoint and mention it here.